### ISSUE TITLE
Expose more options in the PWA site controls notification

### ISSUE THREAD

**Issue by NotWoods** [mozilla-mobile/fenix#7651]
### Why/User Benefit/User Problem
Originally reported by @FluorescentHallucinogen:

Various settings like permissions, addons, and find in page are not exposed to PWAs. It would be helpful to popup a menu when the site controls notification is tapped with many options.

Related equivalent Chrome bug: https://bugs.chromium.org/p/chromium/issues/detail?id=974785

![](https://00e9e64bacfef0530a9e8a8d8a0ee81de75d6021a6e4e9d14b-apidata.googleusercontent.com/download/storage/v1/b/monorail-prod.appspot.com/o/16%2Fattachments%2F57b174a6-a5d1-4780-82c5-ab6b73052422?qk=AD5uMEuze-HOiz81QdDA0p7mQKEEbtr1PDGxzW25lsqTP2BbRZ2uWmMinBQw6rR9nBEHmVN8VCza_HEcwkBCDgF1q5MIojmpnsiqdZPXs-g-ChoqmiwZBrgc311ASeX6tsaKmu1aYcNHUUDv1v8qNEdcaFOd65TSahDhgF82qHNZp-rQJYQi-oNPH1YdEIMQKnPNbuCseDHpiA6nSu9_v-F1IWGxZ5d4ov94ViXnAt48paWSd8S-c7LddFFPXFk_XiNjwt8qXJYzNpBjZ5X5ovFx3qf0ZKM20WftQsWQUrObrkGXFD-bXo-_Q-ofNMjZH-b8c1sBl5leVW9BezfJaEHPqj1hIFg-EOV_GK5m1qM96n9SolBv2aoGC4aM_3l5JsyooJuBiiwfnFCCffXlabtWE8DByIEyZBJ4e_RVxsgs6xzV6gtoLD2zMFPtvwCl_nmt22yQ_msg8UBYY_oXsOeuZiQDX9RN0CxjwWsRpwQnxGZYThGvexRak1L8ZmN-HLuFQZwx2NIY_j_t7sJiOSoc6vMuAnRDZuOzysa7a23JBzFUzMOXQlIMYWGma4qqS0JucQv9HsMfApgQHcWSUUz5ergKzRrDI1kakFccGHfDsjQBNf9RprCEnZTTg5wxMta5vANdJ0n9B6LPZg86tIPPu3kfzQxysnKoaPQbEb9q2oasqmoDkoDY_pnB6wNTVpprkWg6jTMa8TnXnTu9KgtKq1l8WaxhFcINC6GinVNj4Iarmc64mOtffnApQGPxY3Gclu8gxnzI7Ob2oRQL0-KxhXmbcuzH7cY3fji9bfHx4hpC-z3JhBLjXbVTfEU0sNQwNKkLoGbhWCuN6BaX7NR4bnCZp9JWu-RxGWU9eJrgDn0bp0aJEx8)

### What/Requirements
- Expose some way to access all in page browser options

### Acceptance Criteria (how do I know when I’m done?)
- Permissions can be granted/revoked while a PWA is open
- Addon settings can be controlled
- Can search for text in a PWA
- Can share PWA URL

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FNXV2-15169)

**Comment by FluorescentHallucinogen** [comment]
Just noticed that Firefox shows a different persistent notification for TWAs than for PWAs (even without ability to copy the URL).

For PWA:

![firefox-pwa-persistent-notification](https://user-images.githubusercontent.com/7892779/72327850-eadb0900-36c2-11ea-98e8-fb176ca66b04.png)

For TWA:

![firefox-twa-persistent-notification](https://user-images.githubusercontent.com/7892779/72327880-f75f6180-36c2-11ea-8425-55d958fb3009.png)

@NotWoods Since TWAs are also launched in standalone mode, should I open a separate issue for that or will it be fixed as part of this issue?

**Comment by NotWoods** [comment]
It would be part of this issue

**Comment by TitanNano** [comment]
@NotWoods one more acceptance criteria should be "can share PWA"

**Comment by TitanNano** [comment]
@NotWoods I see this is blocked by https://github.com/mozilla-mobile/android-components/pull/5591. Can I pick it up after that PR has landed?

**Comment by NotWoods** [comment]
I want to wait untill browser-menu2 is landed and integrated with the main app and custom tabs. Would you be interested in picking up private mode PWAs instead?

**Comment by TitanNano** [comment]
@NotWoods yes sure. Would that be https://github.com/mozilla-mobile/fenix/issues/7996?

**Comment by NotWoods** [comment]
Yes, I'll update the issue with a mock soon.

**Comment by FluorescentHallucinogen** [comment]
Any news? )

**Comment by cpeterso** [comment]
Moved to bugzilla: https://bugzilla.mozilla.org/show_bug.cgi?id=1807432

Change performed by the [Move to Bugzilla add-on](https://addons.mozilla.org/en-US/firefox/addon/move-to-bugzilla/).
